src/main/java/heronarts/lx/dmx/Dmx16bitModulator.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2023- Justin K. Belcher, Mark C. Slee, Heron Arts LLC
+ *
+ * This file is part of the LX Studio software library. By using
+ * LX, you agree to the terms of the LX Studio Software License
+ * and Distribution Agreement, available at: http://lx.studio/license
+ *
+ * Please note that the LX license is not open-source. The license
+ * allows for free, non-commercial use.
+ *
+ * HERON ARTS MAKES NO WARRANTY, EXPRESS, IMPLIED, STATUTORY, OR
+ * OTHERWISE, AND SPECIFICALLY DISCLAIMS ANY WARRANTY OF
+ * MERCHANTABILITY, NON-INFRINGEMENT, OR FITNESS FOR A PARTICULAR
+ * PURPOSE, WITH RESPECT TO THE SOFTWARE.
+ *
+ * @author Mark C. Slee <mark@heronarts.com>
+ * @author Justin K. Belcher <justin@jkb.studio>
+ */
+
+package heronarts.lx.dmx;
+
+import heronarts.lx.LXCategory;
+import heronarts.lx.modulator.LXModulator;
+
+/**
+ * A modulator using two channels of DMX data for high resolution
+ */
+@LXModulator.Global("DMX 16-bit")
+@LXModulator.Device("DMX 16-bit")
+@LXCategory(LXCategory.DMX)
+public class Dmx16bitModulator extends DmxModulator {
+
+  public Dmx16bitModulator() {
+    this("DMX 16-bit");
+  }
+
+  public Dmx16bitModulator(String label) {
+    super(label);
+    this.channel.setRange(0, LXDmxEngine.MAX_CHANNEL - 1);
```

Comment:
Two universes would be different packets, rights?  So it seems reasonable to force a two-byte data point into a single universe as we're doing here, to prevent the two bytes from being sourced from different frames.